values.yaml

```diff
@@ -0,0 +1,30 @@
+redis:
+  enabled: true
+  auth:
+    enabled: false
+  architecture: standalone
+  master:
+    persistence:
+      enabled: true
+      size: 1Gi  # Adjust the size as needed
+
+onetimesecret:
+  image: 
+    repository: onetimesecret/onetimesecret
+    tag: latest
+  service:
+    type: LoadBalancer
+    port: 3000
+  env:
+    REDIS_URL: "redis://onetimesecret-redis-master.default.svc.cluster.local:6379/0"
```

Comment:
Maybe have the default be the namespace that it's installed in and if a value is provided have it use the value that the user provides? I'm thinking just commenting it out would be sufficient but let the user know it's available as an option easily. What do you think?